src/Claymate.tsx

```diff
@@ -14,6 +14,8 @@ import { exportToGif } from './exportToGif';
 import { exportToHtml } from './exportToHtml';
 import { importFromFile } from './importFromFile';
 import { previewGif } from './previewGif';
+import { saveScene } from './saveScene';
+import { importScene } from './importScene';
```

Comment:
Could we call `exportScenes` & `importScenes`?  (Or `saveScenes` & `loadScenes`)